docs/shared/plugins/intro.md

```diff
@@ -1,58 +1,108 @@
-# Getting Started with Plugins
+# Extending Nx with Plugins
 
-Nx plugins contain [generators](/features/generate-code) and [executors](/concepts/executors-and-configurations) that extend the capabilities of an Nx workspace. They can be shared as npm packages or referenced locally within the same repo.
+Nx's core functionality focuses on task running and understanding your project and task graph. Nx plugins leverage that functionality to enforce best practices, seamlessly integrate tooling and allow developers to get up and running quickly.
 
-{% cards cols="4" %}
+As your repository grows, you'll discover more reasons to create your own plugin
 
-{% link-card title="Use a Plugin" url="#use-a-plugin" /%}
-{% link-card title="Create a Plugin" url="#create-a-local-plugin" /%}
-{% link-card title="Maintain a Published Plugin" url="#maintain-a-published-plugin" /%}
-{% link-card title="Advanced Plugins" url="#advanced-plugins" /%}
+- You can help encourage your coworkers to consistently follow best practices by creating [code generators](/features/generate-code) that are custom built for your repository.
+- You can remove duplicate configuration and ensure accurate caching settings by writing your own [inferred tasks](/concepts/inferred-tasks).
+- For organizations with multiple monorepos, you can encourage consistency across repositories by providing a repository [preset](/extending-nx/recipes/create-preset) and writing [migrations](/extending-nx/recipes/migration-generators) that will help keep every project in sync.
+- You can write a plugin that integrates a tool or framework into Nx and then [share your plugin](/extending-nx/recipes/publish-plugin) with the broader community.
 
-{% /cards %}
+## Create Your Own Plugin
 
-## Use a Plugin
+Get started developing your own plugin with a few terminal commands:
 
-Nx plugins help you scaffold new projects, pre-configure tooling, follow best practices, and modularize your codebase.
+{% side-by-side %}
 
-{% cards cols="3" %}
+```shell {% title="Create a plugin in a new workspace" %}
+npx create-nx-plugin my-plugin
+```
 
-{% card title="Browse Existing Plugins" description="Find a plugin to use" url="/plugin-registry" /%}
-{% card title="Use Task Executors" description="Run operations on your code" url="/concepts/executors-and-configurations" /%}
-{% card title="Generate Code" description="Create or modify code" url="/features/generate-code" /%}
+```shell {% title="Add a plugin to an existing workspace" %}
+npx nx add @nx/plugin
+npx nx g plugin my-plugin
```

Comment:
This will create in `my-plugin` rather than `packages/my-plugin` folder. Either we pass `--directory=packages/my-plugin` here, or maybe in the generator section later we just use `--directory=my-plugin/src/generators/library-with-readme`.

Reply:
I'll make the generator section match this.  Initially, I had the `--directory` flag on the `nx g plugin` command, but it makes the line wrap and look bad.